spec/services/idv/steps/document_capture_step_spec.rb

```diff
@@ -57,75 +57,6 @@
   end
 
   describe '#extra_view_variables' do
-    context 'with acuant sdk upgrade A/B testing disabled' do
-      let(:session_uuid) { SecureRandom.uuid }
```

Comment:
We don't want to delete specs for DocumentCaptureStep until the feature flag has been on for a while and we're deleting the old code.